src/pages/Home.vue

```diff
@@ -78,72 +78,52 @@ form {
   flex-direction: column;
 }
 
-.file {
-  position: relative;
-  display: inline-block;
+input,
+button {
+  font-family: 'Fira Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```

Comment:
This is a kludge. I can't figure out why the input and buttons don't inherit from body font?

Reply:
any ideas @arnellebalane ?

Reply:
@jjlumagbas @emanlodovice That's default behaviour. In the global styles you can add 

```css
input,
button {
  font: inherit;
  color: inherit;
}
```

Reply:
Applied this change in 22e179e